.github/workflows/main.yml

```diff
@@ -103,6 +103,8 @@ jobs:
           - "vai"
           - "vsc_ugent"
           - "wehi"
+          - "dkfz"
```

Comment:
This is alphabetical in order - please put dkfz to `d`.

Reply:
done